deps/build_libs.sh

```diff
@@ -2,22 +2,23 @@
 
 export PREFIX=/opt/lib/${CROSS_COMPILE}
```

Comment:
```suggestion
true  # Dummy command required to prevent first ShellCheck directive from having global scope.

# shellcheck disable=SC2153 # Fix false positive of ShellCheck rule SC2153.
export PREFIX=/opt/lib/${CROSS_COMPILE}
```

Add ShellCheck directive to fix false positive:
https://github.com/koalaman/shellcheck/wiki/directive#disable

Unfortunately since the false positive occurred on the first code line of the script, a directive would be applied to the entire script instead of the specific problematic line as intended:
https://github.com/koalaman/shellcheck/wiki/directive#disable:~:text=Directives%20that%20replace%20or%20are%20immediately%20after%20the%20shebang%20apply%20to%20the%20entire%20script

> Directives that replace or are immediately after the shebang apply to the entire script. Otherwise, they are scoped to the command that follows it (including compound commands like function definitions, loops and case statements).

(not mentioned there is that separating the directive from the shebang with whitespace or comments has no effect on this scoping)

The workaround is to add a "dummy" command above the problematic code line so that it will no longer be the first, as recommended here:

https://github.com/koalaman/shellcheck/wiki/directive#disable:~:text=use%20a%20dummy%20command%20true

> There is no support for scoping a directive to the first structure of the script. In these cases, use a dummy command `true` or `:` and then add directives